docs/guide/service/nlb_ip_mode.md

```diff
@@ -33,5 +33,22 @@ to enable proxy protocol v2, apply the following annotation to your service:
 service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 ```
 
+## Subnet tagging requirements
+You must tag the subnets you wish to attach your ELB to with the following tags. 
+  - For any subnet used by node group. 
+    - **Key:** *kubernetes.io/cluster/$CLUSTER_NAME*
```

Comment:
The cluster tag is optional. In case of ties (for example there are multiple subnets in the same AZ), auto-discovery will prefer the one with the cluster tag. This has been the behavior since EKS 1.19 for in-tree and v2.1.1 release of the lb controller (ref https://docs.aws.amazon.com/eks/latest/userguide/load-balancing.html)